settings.py

```diff
@@ -23,12 +24,12 @@
 
 DEFAULT_VALUES = {
```

Comment:
Add also here 'ids_to_nickname_path' so you can retrieve it calling settings.get()